server/apps/immich/src/api-v1/config/config.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { AdminConfigResponseDto } from './response-dto/admin-config-response.dto';
+import { AdminConfigService } from '@app/admin-config';
+import { AdminConfigEntity } from '@app/database/entities/admin-config.entity';
+
+@Injectable()
+export class ConfigService {
+  constructor(private adminConfigService: AdminConfigService) {}
+
+  async getAllConfig(): Promise<AdminConfigResponseDto> {
+    const config = await this.adminConfigService.getConfig();
```

Comment:
Should the service and the controller use the same dto? That is how all of the other services work.